src/acceleration/test/AccelerationSerialTest.cpp

```diff
@@ -733,23 +647,21 @@ BOOST_AUTO_TEST_CASE(testConstantUnderrelaxationWithGradientWithoutSubsteps)
   mesh::PtrData displacements = std::make_shared<mesh::Data>("dvalues", -1, 1);
   mesh::PtrData forces        = std::make_shared<mesh::Data>("fvalues", -1, 1);
 
-  // //init displacements
-  displacements->values().resize(4);
-  displacements->values() << 1.0, 2.0, 3.0, 4.0;
+  // init displacements
   displacements->requireDataGradient();
-  displacements->gradients().resize(dim, 4);
+  Eigen::MatrixXd displacementGradient(displacements->gradients());
+  displacementGradient.resize(dim, 4);
   for (unsigned int r = 0; r < dim; ++r) {
     for (unsigned int c = 0; c < 4; ++c)
-      displacements->gradients()(r, c) = r + r * c;
+      displacementGradient(r, c) = r + r * c;
   }
-  displacements->setSampleAtTime(windowStart, displacements->sample());
-  // //init forces
-  forces->values().resize(4);
-  forces->values() << 0.2, 0.2, 0.2, 0.2;
+  displacements->setSampleAtTime(windowStart, time::Sample(displacements->getDimensions(), Eigen::Vector4d{1.0, 2.0, 3.0, 4.0}, displacementGradient));
+  // init forces
   forces->requireDataGradient();
-  forces->gradients().resize(dim, 4);
-  forces->gradients().setConstant(-2);
-  forces->setSampleAtTime(windowStart, forces->sample());
+  Eigen::MatrixXd forcesGradient(forces->gradients());
+  forcesGradient.resize(dim, 4);
+  forcesGradient.setConstant(-2);
+  forces->setSampleAtTime(windowStart, time::Sample(forces->getDimensions(), Eigen::Vector4d{0.2, 0.2, 0.2, 0.2}, forcesGradient));
```

Comment:
@fsimonis at this place (and some others) it would be convenient to have the following function:

```
emplaceSampleAtTime(double time, std::initializer_list<double> values, Eigen::MatrixXd gradients)
```

Your opinion?